velox/dwio/parquet/tests/reader/CMakeLists.txt

```diff
@@ -43,16 +34,41 @@ target_link_libraries(
   ZLIB::ZLIB
   ${TEST_LINK_LIBS})
 
-add_executable(velox_dwio_parquet_reader_benchmark ParquetReaderBenchmark.cpp)
+add_library(velox_dwio_parquet_reader_benchmark_lib ParquetReaderBenchmark.cpp)
+target_link_libraries(
+  velox_dwio_parquet_reader_benchmark_lib
+  velox_dwio_parquet_reader
+  velox_dwio_parquet_writer
+  velox_exec_test_lib
+  velox_exec
+  velox_hive_connector
+  Folly::folly
+  ${FOLLY_BENCHMARK}
+  ${TEST_LINK_LIBS})
+
+add_executable(velox_dwio_parquet_reader_benchmark
+               ParquetReaderBenchmarkRunner.cpp)
 target_link_libraries(
   velox_dwio_parquet_reader_benchmark
+  velox_dwio_parquet_reader_benchmark_lib
   velox_dwio_parquet_reader
   velox_dwio_parquet_writer
   velox_exec_test_lib
   velox_exec
   velox_hive_connector
   Folly::folly
-  ${FOLLY_BENCHMARK})
+  ${FOLLY_BENCHMARK}
```

Comment:
Is ${FOLLY_BENCHMARK} still required here?

Reply:
Not required. Removed this